source/includes/azure/_instances.md

```diff
@@ -47,10 +51,14 @@ Attributes | &nbsp;
 `id` <br/>*string* | The id of the instance. This is a canonized id from azure which is the form of /subscriptions/:subscriptionid/resourceGroups/:resourcegroup/providers/Microsoft.Compute/virtualMachines/:instanceName 
 `name` <br/>*string* | The name of the instance
 `machineType`<br/>*string* | The type of machine assigned for this instance
+`numberOfCores`<br/>*int* | The number of cores provisioned for this instance
+`memoryInGB`<br/>*int* | The number of memory in GB provisioned for this instance
```

Comment:
For the memory and number of core, i would specify more the number of memory/core aissgned to the machinetype assigned to the instance. Otherwise, it would mean that we could customize the machine type

Reply:
will do

Reply:
done